assets/js/theme/global/modal.js

```diff
@@ -22,24 +22,26 @@ export const modalTypes = {
     PRODUCT_DETAILS: 'forProductDetails',
     CART_CHANGE_PRODUCT: 'forCartChangeProduct',
     WRITE_REVIEW: 'forWriteReview',
+    SHOW_MORE_OPTIONS: 'forShowMore',
 };
 
+const findRootModalTabbableElements = () => (
+    $('#modal.open')
+        .find(allTabbableElementsSelector)
+        .not('#modal-review-form *')
+        .not('#previewModal *')
+);
+
 const focusableElements = {
-    [modalTypes.QUICK_VIEW]: () => (
-        $('#modal')
-            .find(allTabbableElementsSelector)
-            .not('#modal-review-form *')
-            .not('#previewModal *')
-    ),
+    [modalTypes.QUICK_VIEW]: findRootModalTabbableElements,
     [modalTypes.PRODUCT_DETAILS]: () => (
-        $('#previewModal').find(allTabbableElementsSelector)
-    ),
-    [modalTypes.CART_CHANGE_PRODUCT]: () => (
-        $('#modal').find(allTabbableElementsSelector)
+        $('#previewModal.open').find(allTabbableElementsSelector)
     ),
+    [modalTypes.CART_CHANGE_PRODUCT]: findRootModalTabbableElements,
     [modalTypes.WRITE_REVIEW]: () => (
-        $('#modal-review-form').find(allTabbableElementsSelector)
+        $('#modal-review-form.open').find(allTabbableElementsSelector)
     ),
+    [modalTypes.SHOW_MORE_OPTIONS]: findRootModalTabbableElements,
 };
```

Comment:
Could we unify approach? For me it looks like adding custom attributes for show more and quick view modals looks good

Reply:
discussed in dm - focusableElements object with functions stays at it was